ninja-servlet/src/main/java/ninja/servlet/NinjaServletContext.java

```diff
@@ -140,6 +140,11 @@ public String getRequestUri() {
         return httpServletRequest.getRequestURI();
     }
 
+    @Override
+    public String getRequestParameterString() {
+        return httpServletRequest.getQueryString();
```

Comment:
I think we should write a test that makes sure we do not break the functionality in the future... I am sure there is a way to test this :)

Reply:
Done